internal/api/approve_freight_v1alpha1.go

```diff
@@ -37,7 +38,7 @@ func (s *server) ApproveFreight(
 			return nil, connect.NewError(connect.CodeNotFound,
 				fmt.Errorf("freight %q not found", freightKey.String()))
 		}
-		return nil, connect.NewError(connect.CodeInternal, err)
+		return nil, connect.NewError(connect.CodeUnknown, errors.Wrap(err, "get freight"))
```

Comment:
@devholic I think we have had some miscommunication, which I take responsibility for.

My suggestion wasn't intended to be that handlers should return a connect error with code `connect.CodeUnknown`.

I intended "unknown" in a generic sense. i.e. If a "naked" `error` is returned, I propose the new error-handling logic should make a best effort to figure out what the root cause was and translate that to an appropriate connect error.

imho, the goal should be that when writing a handler, you don't need to do anything special at all in terms of error handling.

Reply:
@krancour Changed to return raw error instead of wrapping error with `Unknown` status code.